### ISSUE TITLE
Is this working?

### ISSUE THREAD

**Issue by cbdeveloper** [leejustin/FGI-Scraper#4]
Is this package still working? Can I get historical data for the Fear and Greed index? How can I use it? The link with instructions doesn't seem to be working.

**Comment by mattlovestech** [comment]
Yup. looks like the instructions are still not working @leejustin
